plugins/modules/katello_content_view_filter.py

```diff
@@ -33,8 +33,17 @@
     - Create and Manage Katello content View filters
 author: "Sean O'Keeffe (@sean797)"
 requirements:
-    - "nailgun >= 0.28.0"
+    - apypie
```

Comment:
I think you can remove this since https://github.com/theforeman/foreman-ansible-modules/pull/469

Reply:
yes!